impl/src/jvmTest/kotlin/org/jetbrains/kotlinx/dl/impl/preprocessing/PreprocessingFinalShapeTest.kt

```diff
@@ -107,7 +107,7 @@ class PreprocessingFinalShapeTest {
                 right = 13
             }
             .toFloatArray { }
-        assertEquals(TensorShape(324, 212, 1), preprocess.getOutputShape(TensorShape(300, 200, 1)))
+        assertEquals(TensorShape(312, 224, 1), preprocess.getOutputShape(TensorShape(300, 200, 1)))
```

Comment:
Why it's here 312 and 224 instead of 212 and 324?

Reply:
Input image height is 300, we pad it with 5 pixels on top and 7 on the bottom. So 300 + 5 + 7 = 312. Input width is 200, we pad is with 11 pixels on the left and 13 pixels on the right -- 200 + 11 + 13 = 224.

The dimensions order in the tensor shape switched, but the padding values stayed the same, so because of this it's not a simple swap of the numbers in the expected result.